src/psyclone/tests/profile_test.py

```diff
@@ -137,10 +137,16 @@ def test_profile_basic(capsys):
                              .children[0].children[0])
 
     new_sched_str = str(new_sched)
+
+    # Note that the first profile node is created using the get_invoke
```

Comment:
You could make `coloured_text` into a static method and then you wouldn't need to create a ProfileNode just to get the coloured text? Alternatively you could use `invoke.schedule.children[0]` as that is a ProfileNode.

Reply:
While I like static, it would be inconsistent with every other use of coloured_text - so I just use your clever invoke.schedule.children[0] idea - thanks!